config/config.go

```diff
@@ -25,6 +26,8 @@ const (
 	IssueTrackerGitlab               = "GITLAB"
 	IssueTrackerPivotal              = "PIVOTAL_TRACKER"
 	IssueTrackerRedmine              = "REDMINE"
+
+	githubAPITokenWarning = "WARNING: Github has API rate limits for all requests which do not contain a token.\nPlease create a read-only access token to increase that limit.\nGo to https://developer.github.com/v3/#rate-limiting for more information."
```

Comment:
Could you run your version of the app locally & provide a screenshot of how this warning looks like?

It would be nice to see the end result being rendered.

Reply:
Absolutely!

<img width="1365" alt="Screen Shot 2020-10-17 at 9 26 38 AM" src="https://user-images.githubusercontent.com/35509748/96339574-3e67f200-105b-11eb-9e55-5d012ede45fb.png">